### PR TITLE
Full walkthrough on first install #303

### PR DESCRIPTION
**Pull request by leogout**
Providing a full installation walkthrough for newcomers.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,14 @@
 conda create -n voila -c conda-forge notebook nodejs
 conda activate voila
 
+# download voila from GitHub
+git clone https://github.com/QuantStack/voila.git
+
+# install JS dependencies and build js assets
+cd js
```
Comment (mkcor) [inline CONTRIBUTING.md]:
```suggestion
cd voila
cd js
```
If you just cloned the repo (line 11), you need to cd into `voila/` first.

**Comment by mkcor** [comment]
Very nice, @leogout !

I believe that, in the general case, a new contributor would fork the project to contribute... But your additions are relevant anyway, falling under "Setting up a development environment."

**Comment by jtpio** [comment]
Thanks @leogout for taking the time to improve the contributing guide.

As @mkcor mentioned, maybe it would be relevant to have a `git clone` command like: `git clone https://github.com/<your-github-username>/voila.git` ?

**Comment by leogout** [comment]
Hello, yes definitely. I've mentionned it in the guide.

**Comment by jtpio** [comment]
Thanks!

**Pull request merged**